packages/example-corvair/package.json

```diff
@@ -0,0 +1,28 @@
+{
```

Comment:
I vote for one of these (no preference between them)
`generator-react_server`
`generator-reactserver`
`generator-redfin-react-server`